src/docs/self-hosted/index.mdx

```diff
@@ -107,6 +107,37 @@ By default Sentry sends anonymous usage statistics to the Sentry team. It helps
 
 You can disable this feature by adding `SENTRY_BEACON = False` into `sentry.conf.py` file.
 
+## Errors-only Self-hosted Sentry
+
+**Disclaimer: This is an experimental beta feature. This means that features and workflows are not completely tested, so use at your own risk!**
+
+Starting from 24.8.0+, users will have the ability to choose between two distinct types of self-hosted Sentry deployments.
+
+**Errors Only**
+
+Errors Only self-hosted offers a small subset of features with an emphasis on minimizing system resources while focusing on what we do best: tackling issues head-on. This lightweight option includes only the following features:
+1. Issues
+2. Alerts
+3. Integrations
+4. Dashboards
+5. Releases
+6. Discover
+
+In order to enable errors only self-hosted, you'll need to update your [.env file](https://github.com/getsentry/self-hosted/blob/master/.env) to include `COMPOSE_PROFILES=errors-only`.
```

Comment:
This applies to existing install too, right? Will it apply in reverse (that is, if I insert this flag then take it off, will that cause issues?)? If not, we should be very clear about that.

Reply:
Yes, applies for existing installs. If you insert this flag and use errors-only self-hosted and want to switch back to feature complete sentry there shouldn't be issues.